benches/multi_party_ecdsa/gg18/keygen.rs

```diff
@@ -24,37 +21,35 @@ mod bench {
         });
     }
     pub fn keygen_t_n_parties(
-        t: usize,
-        n: usize,
+        t: u16,
+        n: u16,
     ) -> (Vec<Keys>, Vec<SharedKeys>, Vec<GE>, GE, VerifiableSS) {
         let parames = Parameters {
             threshold: t,
-            share_count: n.clone(),
+            share_count: n,
         };
-        let party_keys_vec = (0..n.clone())
-            .map(|i| Keys::create(i))
-            .collect::<Vec<Keys>>();
+        let (t, n) = (t as usize, n as usize);
```

Comment:
is redefining absolutely necessary here? are they used into APIs that require `usize`?

Reply:
This is just a handy redefinition to avoid lots of as usize below

Reply:
So it doesn't fix #72?

Reply:
should already fixed

Reply:
![image](https://user-images.githubusercontent.com/4182346/65424585-df7b7d80-de46-11e9-8e4e-4ff5e0344845.png)

stand closer, my friend

Reply:
I'll also fix other usize

Reply:
About `party_index`, since secret shares are taking usize, I'll just keep it here